### DISCUSSION TITLE
Heartbeat or Pulse

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#1089] [Polls]
Currently the words heartbeat and pulse are used in a mixture in MT

**Comment by FJBDev** [comment]
Why not heart rate too ?

Otherwise, I really like the poll function, i didn't know it existed 👍

**Reply by wolfgang-ch** [reply]
I added heart rate but after modifying the text, the already given polls would be discarded but until now there are no polls

**Comment by GurVJ** [comment]
I voted Heart Rate because the abbreviation HR is often used.

**Comment by wolfgang-ch** [comment]
Changed wording in MT 23.5
